website/content/docs/platform/k8s/helm/configuration.mdx

```diff
@@ -134,7 +160,7 @@ and consider if they're appropriate for your deployment.
         injection: enabled
     ```
 
-  - `objectSelector` (`dictionary: {}`) - The selector used by the admission webhook controller to limit what objects can be effected by mutation.
+  - `objectSelector` (`dictionary: {}`) -
```

Comment:
Should this have a similar deprecation notice like `namespaceSelector`?

Reply:
Oops, thanks 👍